tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1517,7 +1517,7 @@ public void RunCommand_Example_2()
                 database.DropCollection(collectionName);
             }
             database.CreateCollection(collectionName);
-            
+
```

Comment:
This file should not have changed as part of this PR.

Reply:
Can you remove this change from the PR?

Reply:
yep